NEWS.md

```diff
@@ -1,6 +1,25 @@
 Regine
 ==========
 
+Version 1.0.0 - 15 March 2021
+---------------------------
+
+**Features** :rocket:
+* [#3](https://github.com/travelping/regine/pull/3) modernize
+* [f4d7c94](https://github.com/travelping/regine/commit/f4d7c94ef27ddc5db40b949fd05b32f47188c367) update CI and build
+* [#4](https://github.com/travelping/regine/pull/4) Add `.github`
+* [#7](https://github.com/travelping/regine/pull/7) Start use `SemVer`
+* [#8](https://github.com/travelping/regine/pull/8) Add `hex` to GH action
```

Comment:
that are hardly feature of the library

Reply:
Then maybe `refactoring` or `improvement`?

Reply:
Moved to `Improvements`.